app/reducers/routes.js

```diff
@@ -13,7 +13,17 @@ export default function reducer(state = { scene: { notifications: [] } }, action
         scene: newScene,
       };
     case GAME_MEMO:
-      const newNotificaitons = state.scene.notifications.concat([action.payload]);
+      newNotifications = state.scene.notifications.concat([action.payload]);
+      newScene = Object.assign({}, state.scene, {
+        notifications: newNotifications
```

Comment:
Optional refactor part 2:
remove line 16 and replace line 18 with:
`notifications: state.scene.notifications.concat([action.payload])`